pageobjects/homepage.js

```diff
@@ -0,0 +1,92 @@
+var page = require('./page');
+var assert = require('assert');
+//var prop = require('./prop');
+
+//Page objects
+var saleButton = '//*[@class="b7afbb84"]'
+var locationBar = '//*[@placeholder="Location"]'
+var locationIndex = '//li[@data-selected="true"]'
+var moreOption = '//span[@aria-label="Collapse expand button"]'
```

Comment:
You can avoid adding a dependency of an element to be `span`. You could use `*` for it

Reply:
Dependency removed